static/app/components/externalIssues/ticketRuleModal.spec.tsx

```diff
@@ -1,79 +1,79 @@
-import type {PropsWithChildren, ReactElement} from 'react';
-import styled from '@emotion/styled';
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {RouterFixture} from 'sentry-fixture/routerFixture';
 
-import {initializeOrg} from 'sentry-test/initializeOrg';
 import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
 import selectEvent from 'sentry-test/selectEvent';
 
 import {addSuccessMessage} from 'sentry/actionCreators/indicator';
 import TicketRuleModal from 'sentry/components/externalIssues/ticketRuleModal';
-import {makeCloseButton} from 'sentry/components/globalModal/components';
+import {
+  makeClosableHeader,
+  makeCloseButton,
+  ModalBody,
+  ModalFooter,
+} from 'sentry/components/globalModal/components';
 import type {IssueAlertRuleAction} from 'sentry/types/alerts';
 import type {IssueConfigField} from 'sentry/types/integrations';
 
 jest.unmock('sentry/utils/recreateRoute');
 jest.mock('sentry/actionCreators/indicator');
 jest.mock('sentry/actionCreators/onboardingTasks');
 
+const defaultIssueConfig = [
+  {
+    name: 'project',
+    label: 'Jira Project',
+    choices: [['10000', 'TEST']],
+    default: '10000',
+    type: 'select',
+    updatesForm: true,
+  },
+  {
+    name: 'issuetype',
+    label: 'Issue Type',
+    default: '10001',
+    type: 'select',
+    choices: [
+      ['10001', 'Improvement'],
+      ['10002', 'Task'],
+      ['10003', 'Sub-task'],
+      ['10004', 'New Feature'],
+      ['10005', 'Bug'],
+      ['10000', 'Epic'],
+    ],
+    updatesForm: true,
+    required: true,
+  },
+];
+
 describe('ProjectAlerts -> TicketRuleModal', function () {
+  const organization = OrganizationFixture();
+  const router = RouterFixture();
+  const onSubmitAction = jest.fn();
   const closeModal = jest.fn();
-  const modalElements = {
-    Header: (p: PropsWithChildren) => p.children as ReactElement,
-    Body: (p: PropsWithChildren) => p.children,
-    Footer: (p: PropsWithChildren) => p.children,
-  };
 
   afterEach(function () {
     closeModal.mockReset();
     MockApiClient.clearMockResponses();
   });
 
-  const doSubmit = async () =>
-    await userEvent.click(screen.getByRole('button', {name: 'Apply Changes'}));
-
   const submitSuccess = async () => {
-    await doSubmit();
+    await userEvent.click(screen.getByRole('button', {name: 'Apply Changes'}));
     expect(addSuccessMessage).toHaveBeenCalled();
     expect(closeModal).toHaveBeenCalled();
   };
 
   const addMockConfigsAPICall = (otherField = {}) => {
     return MockApiClient.addMockResponse({
-      url: '/organizations/org-slug/integrations/1/?ignored=Sprint',
```

Comment:
would try to keep the query param via 

`match: [MockApiClient.matchQuery({ignored: 'Sprint'})],`